### ISSUE TITLE
EMD transpose for tilt series when not recognized

### ISSUE THREAD

**Issue by cryos** [OpenChemistry/tomviz#1970]
EMD doesn't always have the right angle array in place, and then we have no way to get back to a tilt series. We need to be able to offer an operator to perform a reorder as done in  the EMD reader when we ask to reorder/mark as a tilt series manually. It is a generally useful option to go from slices in x (for the C ordered world) to slices in z (in the F ordered world).

**Comment by ercius** [comment]
Here is a snippet for transposing data to the correct order
```python
def transform_scalars(dataset):
    """Define this method for Python operators that 
    transform the input array"""

    from tomviz import utils
    import numpy as np

    # Get the current volume as a numpy array.
    array = utils.get_array(dataset)

    # This is where you operate on your data, here we square root it.
    #result = np.ascontiguousarray(np.transpose(dataset,(1,2,0)))
    result = np.transpose(array,(1,2,0))

    # This is where the transformed data is set, it will display in tomviz.
    utils.set_array(dataset, result)
```

**Comment by psavery** [comment]
Do you think it might be good to add this as an option in the "Set Tilt Angles" operator, which is also used by "Mark Data as Tilt Series"? It can maybe be a spinbox that says "tilt axis" that defaults to 2 (the Z axis). If the user selects a different number, it can swap the axes so that the tilt axis is Z.

![image](https://user-images.githubusercontent.com/9558430/67135383-08452780-f1e7-11e9-9ade-e8714a131117.png)

The separate operator for generally swapping axes will be pretty much the same as Peter's snippet above (except using the new `transform()` API).

**Comment by cjh1** [comment]
This was resolved by #1980
